### ISSUE TITLE
Improvement of README

### ISSUE THREAD

**Issue by diegueus9** [skoczen/django-ajax-uploader#6]
I'm installing this app and was necessary change the code of views for successful test:

```
 from django.shortcuts import render_to_response
 from ajaxuploader.views import AjaxFileUploader
 from django.middleware.csrf import get_token

 def start(request):
     csrf_token = get_token(request)
     return render_to_response(request, 'import.html', {'csrf_token': csrf_token})

 import_uploader = AjaxFileUploader()
```

Should be:

```
from django.middleware.csrf import get_token
from django.shortcuts import render_to_response
from django.template import RequestContext

from ajaxuploader.views import AjaxFileUploader

def start(request):
    csrf_token = get_token(request)
    return render_to_response('import.html',
        {'csrf_token': csrf_token}, context_instance = RequestContext(request))

import_uploader = AjaxFileUploader()
```

**Comment by skoczen** [comment]
Quite right.  I've been using various helper functions for so long, I've completely forgotten the _proper_ way to do it.

Nice catch, I'll have it fixed up shortly.

**Comment by diegueus9** [comment]
This is fixed in pull request #8
